src/test/java/io/strimzi/kafka/bridge/http/ConsumerGeneratedNameTest.java

```diff
@@ -40,33 +41,35 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
-import static io.strimzi.kafka.bridge.http.HttpBridgeTestBase.TEST_TIMEOUT;
+import static io.strimzi.kafka.bridge.Constants.HTTP_BRIDGE;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 
 @ExtendWith(VertxExtension.class)
+@Tag(HTTP_BRIDGE)
 public class ConsumerGeneratedNameTest {
 
     private static final Logger LOGGER = LogManager.getLogger(ConsumerGeneratedNameTest.class);
 
     private static Map<String, Object> config = new HashMap<>();
-    private static final KafkaFacade KAFKA_CLUSTER;
+    private static final StrimziKafkaContainer KAFKA_CLUSTER;
 
     static {
         config.put(KafkaConfig.KAFKA_CONFIG_PREFIX + ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
         config.put(KafkaConsumerConfig.KAFKA_CONSUMER_CONFIG_PREFIX + ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
         config.put(KafkaProducerConfig.KAFKA_PRODUCER_CONFIG_PREFIX + ProducerConfig.MAX_BLOCK_MS_CONFIG, "10000");
         config.put(HttpConfig.HTTP_CONSUMER_TIMEOUT, 5L);
-        KAFKA_CLUSTER = new KafkaFacade();
+        KAFKA_CLUSTER = new StrimziKafkaContainer("latest-kafka-2.4.0");
```

Comment:
hardcoding the version means that we have to update this every time? And 2.4.0 is already old :-)

Reply:
Good catch..replaced by empty constructor where we will guarantee the latest version of Kafka.